kinder/ci/workflows/skew-1.14-on-1.13.yaml

```diff
@@ -7,5 +7,6 @@ summary: |
 vars:
   kubeadmVersion: "{{ resolve `ci/latest-1.14` }}"
   kubernetesVersion: "{{ resolve `ci/latest-1.13` }}"
+  controlPlaneNodes: 2
```

Comment:
it think it be 3 instead of 2 for HA as per our docs and the redundancy factor.

Reply:
@neolit123 considering that the goal of this test is to check `join --control-plane` & `upgrade node`, (not the etcd clustering), two is enough.
However, no problem to change it

Reply:
i've seen instances where users following our guide fail on the 3rd control plane node.
(of course due to a user mistake, for one reason or another).

/lgtm